full-service/src/db/transaction_log.rs

```diff
@@ -1968,4 +1970,63 @@ mod tests {
 
         assert_eq!(tx_log.tx, expected_tx_log.tx);
     }
+
+    #[test]
+    fn test_try_from_vec_output_txo_for_transaction_id() {
+        let mut rng: StdRng = SeedableRng::from_entropy();
+        let root_id = RootIdentity::from_random(&mut rng);
+        let recipient_account_key = AccountKey::from(&root_id);
+
+        let num_txos = 5; // txo vector size
+        let num_loops = 5; // number of times to run test
+        let amount = 77;
+
+        for _ in 1..=num_loops {
+            // run test loop
+
+            let mut output_vec: Vec<OutputTxo> = Vec::new();
+
+            for _ in 0..num_txos {
+                // loop to build vector and test try_from()
```

Comment:
⛏️ I lean pretty hard in the camp of comments should communicate why and code should communicate what
These comments, got formatted funky due to cargo fmt, but also I don't think they provide much benefit
```suggestion
        let txo_vector_size = 5;
        let num_test_runs = 5;
        let amount = 77;

        for _ in 1..=num_test_runs {
            let mut output_vec: Vec<OutputTxo> = Vec::new();

            for _ in 0..txo_vector_size {
```

Reply:
Eh, I don't think it was the cargo formatter.  I just wasn't being very disciplined with my comments, so _your_ comments here are well taken.